### ISSUE TITLE
not collecting tiktok data

### ISSUE THREAD

**Issue by claracluz** [digitalmethodsinitiative/zeeschuimer#19]
Hi!
So, no matter how long I scroll or on whichever page on Tiktok, the number of items on Zeeschuimer keeps being 0. Yesterday, it would only collect from the for you page, but not hashtag pages. today it won't collect at all. I really appreciate any help you can provide.

**Comment by stijn-uva** [comment]
Hi @claracluz , looks like TikTok updated their interface in the past few days so items aren't "recognised" anymore by Zeeschuimer. We'll look into restoring this functionality, thanks for reporting.

**Comment by dale-wahl** [comment]
v1.7.0 currently working for my instance. Possibly they are testing a new interface that has not been rolled out everywhere.

**Comment by stijn-uva** [comment]
@claracluz can you tell us what version of Zeeschuimer you are using? The number is visible at the top of the interface, next to the 'Zeeschuimer' header.

**Comment by claracluz** [comment]
Yea, I'm using v1.7.0 as well. Just checked and it's still not collecting any items on tiktok :(
![image](https://github.com/digitalmethodsinitiative/zeeschuimer/assets/140003014/29daac79-325a-4a12-bb72-17692f6b21d0)

**Comment by stijn-uva** [comment]
Hi @claracluz , this turns out to be hard to address/reproduce because we're not seeing this problem on our side. Have you noticed any changes in the TikTok interface recently? This could indicate that they are 'trying out' a new version of the interface for some, but not all users, which may explain why you're encountering this issue but we are not.

For reference, this is what the For You Page looks like for me in the browser:
<img width="1440" alt="image" src="https://github.com/digitalmethodsinitiative/zeeschuimer/assets/42036349/8f08d12b-5710-48f1-99a6-4faaa5b7089e">

**Comment by claracluz** [comment]
Hi, so sorry for the time it took me to answer! Yea, my for you page looks exactly like that except it's in portuguese. I still don't know why it won't work, but I figured out how to do my research with only the dada I had already collected and saved, so I'm okay with closing this thread. Highly appreciate your time and effort in helping!
![image](https://github.com/digitalmethodsinitiative/zeeschuimer/assets/140003014/ef22439b-fc30-4d31-9cb5-174669ed2cec)
